.github/workflows/SHiELD_parallelworks.yml

```diff
@@ -0,0 +1,45 @@
+name: Compile SHiELD SOLO and run tests
+
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  buildstep:
+    runs-on: [self-hosted]
+    name: SOLO SHiELD build
+    strategy:
+      fail-fast: true
+      max-parallel: 1
+      matrix:
+        runpath: [/pw/storage/PWscripts]
+        runscript: [FV3checkoutStartClusters.py, FV3swStartClusters.py, FV3nhStartClusters.py, FV3hydroStartClusters.py]
+    steps:
```

Comment:
Do each of these run through the cluster spin up part of the API script?

Reply:
They all run through the start up script and just have a different testcmd

Reply:
The first only checks out code, the next 3 each build a different configuration